packages/gatsby/src/schema/node-model.js

```diff
@@ -654,19 +654,14 @@ async function resolveRecursive(
     const gqlField = gqlFields[fieldName]
     const gqlNonNullType = getNullableType(gqlField.type)
     const gqlFieldType = getNamedType(gqlField.type)
-    let innerValue
-    if (gqlField.resolve) {
-      innerValue = await resolveField(
-        nodeModel,
-        schemaComposer,
-        schema,
-        node,
-        gqlField,
-        fieldName
-      )
-    } else {
-      innerValue = node[fieldName]
-    }
+    let innerValue = await resolveField(
+      nodeModel,
+      schemaComposer,
+      schema,
+      node,
+      gqlField,
+      fieldName
+    )
```

Comment:
Was this actually part of the fix? Or a refactor (because it seems to me like the same thing before/after)

Reply:
This is just a related refactor. I moved this condition to the `resolveField` function so as not to duplicate it elsewhere:

```js
if (!gqlField.resolve) {
  return node[fieldName]
}
```